extensions/rke1/nodetemplates/aws/create.go

```diff
@@ -16,7 +17,9 @@ func CreateAWSNodeTemplate(rancherClient *rancher.Client) (*nodetemplates.NodeTe
 	var amazonEC2NodeTemplateConfig nodetemplates.AmazonEC2NodeTemplateConfig
 	config.LoadConfig(nodetemplates.AmazonEC2NodeTemplateConfigurationFileKey, &amazonEC2NodeTemplateConfig)
 
-	cloudCredential, err := aws.CreateAWSCloudCredentials(rancherClient)
+	var cloudCredentialConfig cloudcredentials.CloudCredential
+	config.LoadConfig(cloudcredentials.AmazonEC2CredentialConfigurationFileKey, &cloudCredentialConfig.AmazonEC2CredentialConfig)
```

Comment:
why is the fileKey loaded 2x?

Reply:
The first load is for the nodeTemplateConfig, the second is for the Credentials.